### ISSUE TITLE
Unyson Page Builder Activating Blogs Logs Out Of Wordpress Dashboard

### ISSUE THREAD

**Issue by srochan** [ThemeFuse/Unyson#1339]
I am using WoW-parent template with Unyson. In Page Builder, Pages part of Unyson is working. However if I try enabling Blogs or any other plug in, the page exits and takes me to the Dashboard Login Screen. I request you to help me on this at the earliest.

**Comment by moldcraft** [comment]
Please enable a default WordPress theme and check if the problem persists.

**Comment by Jelena1** [comment]
Exactly same problem has my customer with page builder extension. I tried with default WP theme and only with Unyson plugin active and problem is still there. Can you please help with this issue?

**Comment by moldcraft** [comment]
Please disable other plugins and also check error log.

**Comment by Jelena1** [comment]
Unfortunately customer decided to use another theme and now I don't have access to server anymore. 

I've tested Unyson plugin activated with default WordPress theme, without other plugins. Issue that appeared looked like this: 

When page builder extension is downloaded and when I've attempted to mark "blog posts" in page builder extension's "settings". When the check box for blog posts is marked and when I attempted to save settings it would log me out of dashboard.

I suppose it is something related to server configuration or WP dashboard settings. I've never had such issue with Unyson on any other server.
